<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogColorDenoiseFilter</class>
 <widget class="QWidget" name="DialogColorDenoiseFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color Denoise Filter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_preset">
     <item>
      <widget class="QLabel" name="label_presetTitle">
       <property name="text">
        <string>Preset</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_presetButtons">
       <item>
        <widget class="QRadioButton" name="radioButton_denoiseLow">
         <property name="toolTip">
          <string>Low strength, preserves fine details.</string>
         </property>
         <property name="text">
          <string>Low</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_denoiseMid">
         <property name="toolTip">
          <string>Balanced mode.</string>
         </property>
         <property name="text">
          <string>Medium</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_denoiseHigh">
         <property name="toolTip">
          <string>High strength, stronger smoothing.</string>
         </property>
         <property name="text">
          <string>High</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_kNeighbors">
       <property name="text">
        <string>k neighbors</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBox_kNeighbors">
       <property name="minimum">
        <number>8</number>
       </property>
       <property name="maximum">
        <number>64</number>
       </property>
       <property name="value">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_strength">
       <property name="text">
        <string>Strength</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBox_strength">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_luminanceStrength">
       <property name="text">
        <string>Luminance strength</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBox_luminanceStrength">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_radiusFactor">
       <property name="text">
        <string>Radius factor</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_radiusFactor">
       <property name="minimum">
        <double>2.000000000000000</double>
       </property>
       <property name="maximum">
        <double>6.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>4.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_iterations">
       <property name="text">
        <string>Iterations</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spinBox_iterations">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>3</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_preserveLuminance">
     <property name="text">
      <string>Preserve luminance (keep edges)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_folder">
     <item row="0" column="0">
      <widget class="QLabel" name="label_folder">
       <property name="text">
        <string>Folder</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_folder">
       <property name="placeholderText">
        <string>Enter output folder</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="toolButton_folder">
       <property name="text">
        <string>Select Out Folder</string>
       </property>
       <property name="icon">
        <iconset resource="../../../openscantools.qrc">
         <normaloff>:/icons/100x100/open_project.png</normaloff>:/icons/100x100/open_project.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_openFolderAfterExport">
     <property name="text">
      <string>Open folder after export</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_buttons">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../openscantools.qrc"/>
 </resources>
 <connections/>
</ui>
